### DISCUSSION TITLE
If time between frist URL and redirected URL includes in metrics

### DISCUSSION THREAD

**Discussion by AlinaKIRIC** [GoogleChrome/lighthouse-ci#643] [Q&A]
Hi! I have one question, hope you can clarify me that.

I start URL in lighthouse where we have redirect to other URL according to location. 
Frist URL was executed several seconds, after that we have redirect to second URL. The main question how that influence on metrics. 
Lighthouse start to analize time from first URL or when we have redirect Lighthouse start to check dom content load, full load e.t.c from the second(redirected) URL starting?

If Lighthose start from second URL where I can find time from first URL till redirected URL?(to understand how customers see that, how slow or fast it works)

**Comment by patrickhulce** [comment]
Good question! It's a complicated area that is actually handled differently depending on your throttlingMethod.

See https://github.com/GoogleChrome/lighthouse/issues/8984 for details, but if you are using `throttlingMethod: 'simulate'` (the default, if you haven't changed anything) then the full cost of the navigation, including redirects, should be included.
